docs/02-app/01-building-your-application/02-data-fetching/02-server-actions-and-mutations.mdx

```diff
@@ -934,7 +934,6 @@ export default function Page() {
 ```
 
 ```jsx filename="app/page.js" switcher
-'use client'
 export default function Page() {
```

Comment:
unrelated to your changes but this function should be async if there's an await inside of it